libs/ui/catalog/src/lib/language-switcher/language-switcher.component.html

```diff
@@ -5,6 +5,8 @@
   [selected]="currentLang"
   ariaName="languages"
   [showTitle]="false"
-  [extraBtnClass]="'flex justify-items-center border border-white !pl-2 !py-1'"
+  [extraBtnClass]="
+    'flex justify-items-center border border-white bg-primary text-white !pl-2 !py-1'
+  "
```

Comment:
Sorry, I think it should actually be transparent, as it picks up the header background color in the mockups.

Reply:
Okay, that's what I thought too, primary doesnt look very good